beacon-chain/rpc/eth/rewards/structs.go

```diff
@@ -0,0 +1,85 @@
+package rewards
+
+import (
+	"encoding/json"
+	"strconv"
+
+	"github.com/pkg/errors"
+	types "github.com/prysmaticlabs/prysm/v4/consensus-types/primitives"
+)
+
+type BlockRewardsResponse struct {
+	Data                *BlockRewards `json:"data"`
+	ExecutionOptimistic bool          `json:"execution_optimistic"`
+	Finalized           bool          `json:"finalized"`
+}
+
+type BlockRewards struct {
```

Comment:
Why do we need this type - do we never need to read unmarshal a value into BlockRewards? Could this be a bit simpler if the handler used `BlockRewardsJson` and explicitly converted values to strings, so the value can just be passed to standard json.Marshal?